### PR TITLE
Marcos

### PR DESCRIPTION
**Pull request by marwernicke**
First pull

### CODE REVIEW

File: user.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# my files
+from connection import conect
+from client_functions import orders
+from Strategies import rolling_high
+
+
+class user():
+    def __init__(self,API_KEY, API_SECRET):
+        self.bfx = conect(API_KEY, API_SECRET)
+        #self.wallet = read_wallet()
```
Comment (ezewer) [inline user.py:12]: Why is this commented

File: client_functions/orders.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  if response != None:
+      print ("NEW Offer: ", response.notify_info)
+      id = vars(response.notify_info)['id']
+      return(id)
+  else:
+      return(None)
+
+async def cancel_all_offers():
+    remove = []
+    for offer in billetera.bot.offers:
```
Comment (ezewer) [inline client_functions/orders.py:16]: billetera or wallet? why using Spanish and English mixed?

File: user.py (hunk 1 above)
Comment (marwernicke) [reply user.py:12]: its an old version, the function wasnt available yet

File: client_functions/orders.py (hunk 2 above)
Comment (marwernicke) [reply client_functions/orders.py:16]: Es codigo viejo eso, nose porque esta ahi. Era en español para no confundirlo con el objeto Wallet de bfx

**Pull request closed without merging**